### DISCUSSION TITLE
油猴今天突然弹出本脚本的跨域请求页面，要选择总是允许吗，这个跨域请求实现什么的

### DISCUSSION THREAD

**Discussion by zhougjstudy** [the1812/Bilibili-Evolved#3755] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/64772517/196960747-84378779-7f91-42b3-8d69-9f19cf7c14b1.png)
油猴今天突然弹出本脚本的跨域请求页面，要选择总是允许吗，这个跨域请求实现什么的

**Comment by the1812** [comment, marked as answer]
下载功能的，我声明过这个域名了，不知道为什么还会弹提示
